Directory.Build.targets

```diff
@@ -110,6 +110,8 @@
     <PackageReference Update="MassTransit.EntityFrameworkCore" Version="$(MassTransitVersion)" />
 
     <PackageReference Update="OpenTelemetry" Version="1.3.1" />
+
+    <PackageReference Update="NUlid" Version="1.3.2" />
```

Comment:
I understand this is an alternative implementation used only for testing purposes?

Reply:
Yes